x-pack/test/functional/apps/ml/permissions/read_ml_access.ts

```diff
@@ -0,0 +1,426 @@
+            await ml.jobTable.clickOpenJobInSingleMetricViewerButton(adJobId);
+            await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+
+            await ml.testExecution.logTestStep('should pre-fill the AD job selection');
+            await ml.jobSelection.assertJobSelection([adJobId]);
+
+            await ml.testExecution.logTestStep('should pre-fill the detector input');
+            await ml.singleMetricViewer.assertDetectorInputExsist();
+            await ml.singleMetricViewer.assertDetectorInputValue('0');
+
+            await ml.testExecution.logTestStep('should input the airline entity value');
+            await ml.singleMetricViewer.assertEntityInputExsist('airline');
+            await ml.singleMetricViewer.assertEntityInputSelection('airline', []);
+            await ml.singleMetricViewer.selectEntityValue('airline', 'AAL');
+
+            await ml.testExecution.logTestStep('should displays the chart');
+            await ml.singleMetricViewer.assertChartExsist();
+
+            await ml.testExecution.logTestStep('should display the annotations section');
+            await ml.singleMetricViewer.assertAnnotationsExists('loaded');
+
+            await ml.testExecution.logTestStep('should display the anomalies table with entries');
+            await ml.anomaliesTable.assertTableExists();
+            await ml.anomaliesTable.assertTableNotEmpty();
+
+            await ml.testExecution.logTestStep('should not display the anomaly row action button');
+            await ml.anomaliesTable.assertAnomalyActionsMenuButtonNotExists(0);
+
+            await ml.testExecution.logTestStep(
+              'should display the forecast modal with disabled run button'
+            );
+            await ml.singleMetricViewer.assertForecastButtonExists();
+            await ml.singleMetricViewer.assertForecastButtonEnabled(true);
+            await ml.singleMetricViewer.openForecastModal();
+            await ml.singleMetricViewer.assertForeCastModalRunButtonEnabled(false);
+            await ml.singleMetricViewer.closeForecastModal();
+          });
+
+          it('should display elements on Anomaly Explorer page correctly', async () => {
+            await ml.testExecution.logTestStep('should open AD job in the anomaly explorer');
+            await ml.singleMetricViewer.openAnomalyExplorer();
+            await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+
+            await ml.testExecution.logTestStep('should pre-fill the AD job selection');
+            await ml.jobSelection.assertJobSelection([adJobId]);
+
+            await ml.testExecution.logTestStep('should display the influencers list');
+            await ml.anomalyExplorer.assertInfluencerListExists();
+            await ml.anomalyExplorer.assertInfluencerFieldListNotEmpty('airline');
+
+            await ml.testExecution.logTestStep('should display the swimlanes');
```

Comment:
Nit - we use two words for `swim lanes` now

Reply:
Fixed in 9fd1368